fcrepo-http-api/src/main/java/org/fcrepo/http/api/services/HttpRdfService.java

```diff
@@ -161,6 +164,25 @@ public RdfStream bodyToInternalStream(final String extResourceId, final InputStr
         return fromModel(model.getResource(idTranslator.toInternalId(extResourceId)).asNode(), model);
     }
 
+    /**
+     * Takes a PATCH request body and translates any subjects and objects that are in the domain of the repository
+     * to use internal IDs.
+     * @param extResourceId the internal ID of the current resource.
+     * @param requestBody the request body.
+     * @param idTranslator an ID converter for the current context.
+     * @return the converted string.
+     */
+    public String patchRequestToInternalString(final String extResourceId, final String requestBody,
```

Comment:
The javadoc says "extResourceId" is the internal ID... what does the "ext" stand for?

Reply:
I was stealing from the other functions here, but it appears that they might be mis-named as well.